src/wix/test/WixToolsetTest.CoreIntegration/BundleExtractionFixture.cs

```diff
@@ -23,7 +23,7 @@ public void CanExtractBundleWithDetachedContainer()
                 var exePath = Path.Combine(baseFolder, @"bin\test.exe");
                 var extractFolderPath = Path.Combine(baseFolder, "extract");
                 var baFolderPath = Path.Combine(extractFolderPath, "BA");
-                var attachedContainerFolderPath = Path.Combine(extractFolderPath, "WixAttachedContainer");
+                var msiPath = Path.Combine(extractFolderPath, "_1", "test.msi");
```

Comment:
Interesting. What caused this to change? `WixAttachedContainer` is a better name than `_1` but I'm missing something.